lib/rest2/index.js

```diff
@@ -48,8 +48,15 @@ class MockRESTv2Server extends MockServer {
     _keys(RESTV2_MOCK_METHODS).forEach((route) => {
       const auth = route.split('/')[2] === 'auth'
 
+      const routeData = RESTV2_MOCK_METHODS[route]
+      let routeKey = routeData
+      let routeType = auth ? 'post' : 'get'
+      if(Array.isArray(routeData)) {
```

Comment:
hmm in which cases this is array?

Reply:
Removed the code, as it was needed for internal use only endpoints which where removed.